govspeak.gemspec

```diff
@@ -42,7 +42,6 @@ library for use in the UK Government Single Domain project'
   s.add_development_dependency "minitest", "~> 5.14"
   s.add_development_dependency "pry-byebug"
   s.add_development_dependency "rake"
-  s.add_development_dependency "rubocop-govuk", "~> 4.2.0"
+  s.add_development_dependency "rubocop-govuk", "~> 4"
```

Comment:
I think you might want to actually set this as a fixed dependency of "4.2.0" (without the `~>`) as it can be annoying when the linting of this differs depending where you're running it. 

For example, if you install locally and have rubocop-govuk 4.0 you may pass the linting locally but then have it fail in CI as it's installed 4.2.